### ISSUE TITLE
Not woking in Android 8.0

### ISSUE THREAD

**Issue by yakobosagung** [nielscup/ShareFile#11]
Please help, i using your plugin in xamarin form. Your plugin success in Android 7.11 but when i run my application in Android 8.0. There aren't action.

**Comment by yakobosagung** [comment]
this is message from output

Plugin.ShareFile: ShareLocalFile Exception: Android.OS.FileUriExposedException: file:///storage/emulated/0/Android/data/com.companyname.myfile/files/Download/filethis.pdf exposed beyond app through ClipData.Item.getUri()

**Comment by Lelelo1** [comment]
You are right about android 8.0 and above does not show the share interface. In api 23 and api 25 I am unable to share. On these versions the interfaces show - but the files don't show up on google drive for instance.
Are you sure the file is shared to the specified location in Android 7.11 by controlling it at the location?

**Comment by stepkillah** [comment]
[PR ](https://github.com/nielscup/ShareFile/pull/10)to fix this issue
